icu4c/source/i18n/number_symbolswrapper.cpp

```diff
@@ -4,6 +4,8 @@
 #include "number_microprops.h"
 #include "unicode/numberformatter.h"
 
+#if !UCONFIG_NO_FORMATTING
```

Comment:
I think what we normally do is

```
// © 2020 and later: Unicode, Inc. and others.
// License & terms of use: http://www.unicode.org/copyright.html

#include "unicode/utypes.h"

#if !UCONFIG_NO_FORMATTING

#include "number_microprops.h"
#include "unicode/numberformatter.h"

// ...

#endif /* #if !UCONFIG_NO_FORMATTING */
```

Basically, put the opening `#if !UCONFIG_NO_FORMATTING` before any other imports besides utypes.h